$NetBSD: patch-Makefile.am,v 1.2 2026/10/19 07:30:00 tm Exp $

thirdparty deps are provided by pkgsrc. debian deps are not
required in pkgsrc.

--- Makefile.am.orig	2026-07-06 19:52:13.000000000 +0000
+++ Makefile.am
@@ -5,7 +5,7 @@ AUTOMAKE_OPTIONS =  foreign
 
 SUFFIXES = .1 .man .pm
 
-SUBDIRS = lib thirdparty debian
+SUBDIRS = lib
 
 BIN = bin/@PACKAGE@ bin/znapzendzetup bin/znapzendztatz
 PM  = @PERL_MODULES_EXTRA_DIST@
